codex/blocktype.nim

```diff
@@ -147,24 +147,25 @@ func new*(
     cid: cid,
     data: @data).success
 
-func new*(
+proc new*(
```

Comment:
Any reason to change to a proc, func has no side effects and are overall "safer", I don't see anything that would trigger a side effect here?

Reply:
Fixed in #566 , it's `func` once again